### DISCUSSION TITLE
Add informational quickfix when an import could not be resolved

### DISCUSSION THREAD

**Discussion by karrtikr** [microsoft/pylance-release#3297] [Enhancement]
![image](https://user-images.githubusercontent.com/13199757/188751345-979190f8-e9c8-44c6-af37-700bc91a69f9.png)

During a user study for testing getting started experience, 3 out of 4 users got stuck at this step. They went searching outside the tool (VSCode) on how to resolve the issue, which is when they got lost as they weren't sure if the solution applies to them.

It would be nice if we could provide an internal link here like Typescript does:

![image](https://user-images.githubusercontent.com/13199757/188752365-62081e43-e8fe-46ab-a733-44ce76819846.png)

about how one can generally resolve these errors. Same as https://github.com/microsoft/pylance-release/issues/2483.

cc/ @cwebster-99 @karthiknadig

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Reply by karrtikr** [reply]
Thanks. Although note upvotes here may not be indicative of how impactful this could be, as most of the getting started experience users are novices and do not report or track issues. This is for retention of those users.

**Comment by luabud** [comment]
I like this idea, we can add a section on our docs @ https://code.visualstudio.com/docs/python/editing#_troubleshooting-intellisense explaining when this happens and how one could fix it, and link it from the quick fix.

**Reply by luabud** [reply]
@karrtikr  oh actually talking about it with the team, given this is an UI element, wouldn't this need to be handled on the Python extension side?

**Reply by karrtikr** [reply]
We don't control the quickfix so I don't see how we can contribute, @karthiknadig can we?

**Reply by luabud** [reply]
Sorry, you're right, I think it needs to be done in the Pylance side.  I opened https://github.com/microsoft/pylance-release/issues/3338 to track this and added to our roadmap

**Comment by heejaechang** [comment]
this requires support from LSP. opened an issue on LSP - https://github.com/microsoft/vscode-languageserver-node/issues/1076

**Comment by heejaechang** [comment]
link to how TS does this - https://github.com/microsoft/vscode/blob/main/extensions/typescript-language-features/src/languageFeatures/refactor.ts#L266
